src/Uno.UI/UI/Xaml/UIElement.skia.cs

```diff
@@ -354,12 +354,19 @@ internal virtual void OnArrangeVisual(Rect rect, Rect? clip)
 					roundedRectClip = LayoutRound(roundedRectClip);
 				}
 
-				Visual.Clip = Visual.Compositor.CreateRectangleClip(
+				var compositionClip = Visual.Compositor.CreateRectangleClip(
 					top: (float)roundedRectClip.Top,
 					left: (float)roundedRectClip.Left,
 					bottom: (float)roundedRectClip.Bottom,
 					right: (float)roundedRectClip.Right
 				);
+
+				if (Clip.Transform is { } clipTransform)
```

Comment:
Should we add the transformation as a parameter of the method on skia instead? It sounds weird to me that the rect is read by caller but transform is read independently here.